pkg/scheduler/framework/runtime/framework.go

```diff
@@ -108,17 +108,17 @@ type extensionPoint struct {
 
 func (f *frameworkImpl) getExtensionPoints(plugins *config.Plugins) []extensionPoint {
```

Comment:
can we change `extensionPoint` to not hold pointers? evaluate if it simplifies things, otherwise leave it.

Reply:
should be fine, just wait for all test results.